### ISSUE TITLE
Problem with containers and full width blocks

### ISSUE THREAD

**Issue by PursuitDream** [twentytwoo/elk-css#1]
Container doens't allow to create full width blocks under or inside it.

![2017-04-14_21-15-06](https://cloud.githubusercontent.com/assets/13126987/25042092/9f5f63ae-2158-11e7-9e6b-41b536b2b674.png)

**Comment by twentytwoo** [comment]
Typically you're only supposed to use one container div, you can create "a full spanned div under container", via

```html
<div class="container">
	<div class="column">
		<div class="stretch">
			<!-- my content here-->
		</div>
		<br />
		<div class="stretch">
			<!-- my footer content here-->
		</div>
	</div>
</div>
```
Thus two fully spanning divs are created, I'll add ability to add two containers when I have the time later tonight.

**Comment by twentytwoo** [comment]
Closed via b674ded44d8dde21b601bc2419d94bb83d28d479
